docs/api-and-integration/metadata/jira.md

```diff
@@ -3,69 +3,69 @@ title: Jira Issue Tracking Integration
-    - Release Note Prefix - Specifying a 'release note prefix' tells Octopus to search through your Jira issue description for that value. If it finds a match, then it will use the issue description as the release note; otherwise, it will use the issue title. See [Release Notes](/docs/api-and-integration/metadata/release-notes-templates.md) for more information.
+    - Jira username/password: Octopus can retrieve Jira Issue descriptions if you specify your Atlassian Cloud username and an API token. You can create an API token from an Atlassian account in the 'Security' area.
+    - Release Note Prefix: Specifying a 'release note prefix' tells Octopus to search through your Jira issue description for that value. If it finds a match, then it will use the issue description as the release note; otherwise, it will use the issue title. See [Release Notes](/docs/api-and-integration/metadata/release-notes-templates.md) for more information.
 
-1. Set the Octopus Server URL in Octopus Deploy
+1. Set the Octopus Server URL in Octopus Deploy.
 
-    Navigate to the {{Configuration,Nodes}} page and ensure you have set the Server URI field to your Octopus Server's base URL. i.e., https://my-company.octopus.app/ or https://my-company-internal-name/
+    Navigate to the **{{Configuration,Nodes}}** page and ensure you have set the Server URI field to your Octopus Server's base URL. i.e., https://my-company.octopus.app/ or https://my-company-internal-name/
 
-1. Configure the Octopus Deploy plugin for Jira
+1. Configure the Octopus Deploy plugin for Jira.
 
-    Navigate to the **{{Configuration,Settings,Jira Issue Tracker}}** page in Octopus and copy the **Octopus InstallationID** and add it to Jira App configuration.
+    Navigate to the **{{Configuration,Settings,Jira Issue Tracker}}** page in Octopus, copy the **Octopus InstallationID**, and add it to Jira App configuration.
 
-1. Update your Octopus environment settings
+1. Update your Octopus environment settings.
 
-    Navigate to **{{Infrastructure,Environments}}** to map your Octopus environments to Jira environment types. This is required so Jira is able to understand Octopus environments and track issue progress. 
-    
-    Note: Jira environment types are a fixed list that cannot be edited. 
+    Navigate to **{{Infrastructure,Environments}}** to map your Octopus environments to Jira environment types. This is required so Jira can understand Octopus environments and track issue progress.
 
-The end result of this integration is to provide Jira with updates on the progress of Jira issues (work items) through the pipeline.
+    Note: Jira environment types are a fixed list that cannot be edited.
+
+When configured this integration will provide Jira with updates about the progress of Jira issues (work items) through the pipeline.
```

Comment:
For the sake of this audience, I don't know that we need to include the phrase (work items).

Reply:
Agreed.